migrations/1_migration.js

```diff
@@ -119,6 +119,7 @@ module.exports = async function(deployer) {
         await lsr.deposit(user1, offset);
         await csr.deposit(user1, offset + 1);
       }
+      break;
```

Comment:
\>sneaking unrelated changes into the commit  
stop doing this lol

(But it's a good change, so I'll once again allow it for now.)